src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+        }
+
+        for (Machine machine : machines) {
```

Comment:
```suggestion
```

Reply:
please check the code, I fixed the errors